src/utils.cxx

```diff
@@ -449,15 +519,16 @@ void LibraryFinder::EvalSearchPaths() {
     if (!this->evald_search_paths.empty())
         return;
     for (std::string var: this->search_vars) {
-        std::string envVal = getenv(var.c_str());
+        std::string envVal = GetSpackEnv(var.c_str());
         if (!envVal.empty())
             this->evald_search_paths[var] = split(envVal, ";");
     }
 }
 
 std::string LibraryFinder::Finder(const std::string &pth, const std::string &lib_name) {
     WIN32_FIND_DATAW findFileData;
-    HANDLE hFind = FindFirstFileW(ConvertAnsiToWide(pth).c_str(), &findFileData);
+    std::string searcher = pth + "\\*";
+    HANDLE hFind = FindFirstFileW(ConvertAnsiToWide(searcher).c_str(), &findFileData);
```

Comment:
Can you add a docstring to this method? This looks like a glob, what was it doing before? Was this a bug, or did the intended input change?

Reply:
Intended input changed, will document. 

Originally I was just going to pass in a path to a library file, now I've decided I'd rather it process a directory of files.